### ISSUE TITLE
Add QM9 datasets

### ISSUE THREAD

**Issue by yuehhua** [yuehhua/GraphMLDatasets.jl#29]
Support Qm9 dataset

> Ref: http://quantum-machine.org/datasets/ or https://pytorch-geometric.readthedocs.io/en/latest/modules/datasets.html

**Comment by CaiYitao** [comment]
do you mean we will be able to load the GEOM datasets like qm9.pkl  iso17 or drug, in GraphMLDatasets?

**Comment by yuehhua** [comment]
Yes, you can list datasets you want GraphMLDatasets to support.
GraphMLDatasets will give graph, node features, etc., separately by GraphMLDatasets's API, e.g. `graphdata(QM9())`, `node_feature(QM9())`.
So, you can organize them in your own way, instead of bounded by `Data` class from torch geometric.

**Comment by CaiYitao** [comment]
At the moment I am dealing with .sdf and .mol kind of datasets which can be loaded by MolecularGraph.jl, but the GEOM datasets(including qm9.pkl, iso17.pkl, drug.pkl,  these datsets are .pkl file) from that ConfGF package can not be loaded.

**Comment by yuehhua** [comment]
So, you need qm9, iso17, and drug?

**Comment by CaiYitao** [comment]
yeah, it would be great if we have those, the original dataset [GEOM](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/JNGTDF) from Harvard, it contains qm9 and drug. 
after you add them, can we open those qm9.pkl, iso17.pkl file in Julia by Pickle.jl or by GraphMLDatasets.jl?
Thank you!

**Comment by yuehhua** [comment]
It's a bite tricky to support the processed version. Does the original dataset work in your condition?

**Comment by CaiYitao** [comment]
I have not tried original data yet, Maybe  I will try to use .sdf  dataset only. 
Thank you!

**Comment by CaiYitao** [comment]
I noticed that you also have GeometricFlux,  have you considered using Optimal Transport to connect Geometric deep learning with Neural ODE?  I found the course of  [optimal transport](http://conformalgeometry.org/lectures/2021/OT/) is very inspiring, so would like to share with you : )
